live-examples/html-examples/text-content/main.html

```diff
@@ -0,0 +1,7 @@
+<header>
+    <img class='logo' src='/media/examples/lizard.png'>
+    <span>Gecko Goodness</span>
+</header>
+<main>
```

Comment:
For the semantic value of this to be correctly identified in IE11 and lower one needs to add `role="main"`. I reckon it would be good to illustrate that here.